src/js/utils/language.js

```diff
@@ -87,7 +87,8 @@ export function getLanguage() {
     return language || navigator.language || 'en';
 }
 
-export const translatedLanguageCodes = ['ar', 'da', 'de', 'el', 'es', 'fi', 'fr', 'he', 'id', 'it', 'ja', 'ko', 'nl', 'no', 'oc', 'pt', 'ro', 'ru', 'sl', 'sv', 'th', 'tr', 'vi', 'zh'];
+// TODO: Deprecate "no", keep "nn" and "nb"
```

Comment:
why deprecate "no" ? I did a quick search and it seems that "nn" and "nb" are dialects of "no", but it seems like "no" is still in use

Reply:
From a software perspective, `nn` and `nb` are languages but `no` is considered a macrolanguage.

![image](https://user-images.githubusercontent.com/2903955/100929669-fd665880-34b5-11eb-8ee8-5e1cc35cd88e.png)

In the case of `no`, when specified as the value for `lang` in an HTML attribute, a page's language defaults to `nb`. Developers familiar with this behavior might find it strange that `no` would be handled as a separate language altogether, and it would be inaccurate to say we have translations for every language contained in the superset of `no`.

Makes more sense to have one translation for each "dialect" but to set it to an expected default (ie. `nb` if we were to follow Chrome's conventions) when the generic `no` code is used.

Reply:
makes sense thanks!